pkg/common/volume_operations.go

```diff
@@ -91,7 +95,7 @@ func NewVolumeOperationsImpl(k8sClient *k8s.KubeClient, logger *logrus.Logger, c
 // Receives golang context and api.Volume which is Spec of Volume CR to create
 // Returns api.Volume instance that took the place of chosen by SearchAC method AvailableCapacity CR
 func (vo *VolumeOperationsImpl) CreateVolume(ctx context.Context, v api.Volume) (*api.Volume, error) {
-	defer vo.metrics.EvaluateDuration("CreateVolume", time.Now())
+	defer vo.metrics.EvaluateDuration("CreateVolume")
```

Comment:
You should add function call at the end of the string
```
defer vo.metrics.EvaluateDuration("CreateVolume")()
```

Reply:
Thanks! Commited the wrong version

Reply:
Fixed